src/sqlittaaja/__init__.py

```diff
@@ -15,8 +15,10 @@ def main():
     init_script = config["answer"].get("initialize", "")
     answer = config["answer"]["exercise"]
     exercises = extract(config["exercise"]["path"])
+    # Treshold default 90% if not in config.toml
+    treshold_pct = config["check_options"].get("treshold_pct", 0.9)
```

Comment:
Remember to validate that `check_options` exists in the config.

Reply:
Forgot to git add config.py